.github/workflows/osd-increment-plugin-versions.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: Increment OpenSearch Dashboard Plugins Version
+
+on:
```

Comment:
Please add cron too:
```
 schedule:
   - cron: 0 0 * * *
```

Reply:
Sayali initially once merged wanted to run the entire workflow with `workflow_dispatch` before we add the `cron`daily.

Reply:
Sounds good! Wondering if we there is a better way to get all plugins' list from somewhere. Either manifest or https://github.com/opensearch-project/opensearch-plugins

Reply:
Sure Sayali, let me take a look at it as next step once merged, If I can get that list for `.meta` file and convert to a repo matrix.